infra/Makefile

```diff
@@ -8,3 +12,6 @@ init:
 
 tf-apply:
 	cd $(ENVIRONMENT) && terraform apply -var-file="secret.tfvars"
+	
+frontend-deploy:
+	cd $(FRONTEND_SRC) && npm run build && gsutil cp -r $(FRONTEND_BUILD_FOLDER)/* gs://$(TF_VAR_frontend_bucket_name)
```

Comment:
It feels like the infra "module" is overstepping its bounds by going to the frontend module and running its scripts. It would be a bit odd if a frontend dev has to run scripts from `infra/` to deploy a new version, especially since deployment isn't typically an infra concern.

Maybe this suggests that we should put this script in a top-level Makefile that can see all the other modules, or this script should be in the frontend Makefile (but then how would it see the bucket from terraform? Hmm).

WDYT?

Reply:
Would it make sense to separate the frontend specific commands into the frontend Makefile? That would bring in a dependency (the infra commands need to be executed before the frontend make commands), but I think it separates concerns well.

Reply:
> Would it make sense to separate the frontend specific commands into the frontend Makefile?

Yes, I think so. This would be a much cleaner separation of concerns.

> That would bring in a dependency (the infra commands need to be executed before the frontend make commands)

I'm not sure I follow this line of thinking. What are you envisioning?

The idea in my head doesn't seems to have the dependency you're thinking of:
- `frontend` would have a script that builds and outputs to `frontend/build/` (basically `npm build`)
- `infra/` would have a script that creates the frontend bucket (i.e. by running `terraform apply`)
- the top level makefile would know how to get the frontend bucket name and use it to deploy a new version of the frontend